tools/nemo_forced_aligner/align.py

```diff
@@ -51,6 +51,8 @@
         If the ASR model is a QuartzNet model, its downsample factor is 2.
         If the ASR model is a Conformer CTC model, its downsample factor is 4.
         If the ASR model is a Citirnet model, its downsample factor is 8.
+    simulate_cache_aware_streaming: only valid for cache aware streaming conformer model, set true to do somulate
```

Comment:
Is there any docs on these aligners?

Reply:
Has the docs updated with this new feature and how users may use it?
Is there any plan to add these docs to the main docs of nemo?